Dockerfile

```diff
@@ -1,5 +1,42 @@
-FROM ubuntu:18.04
+FROM amsterdam/python:3.8-buster as app
+MAINTAINER datapunt@amsterdam.nl
 
-EXPOSE 8089
-RUN apt-get update && apt-get install -y python3-pip
-RUN pip3 install locust
+ENV PYTHONUNBUFFERED 1
+ARG AUTHORIZATION_TOKEN
+ARG GET_AUTHORIZATION_TOKEN
+ARG SECRET_KEY
+
+EXPOSE 8000
+
+RUN mkdir -p /static && chown datapunt /static
+
+COPY requirements.txt /app/
+RUN pip install -r requirements.txt
+
+# TODO: Remove requirements_dev in favor of a multistage dockerfile
+COPY requirements_dev.txt /app/
+RUN pip install -r requirements_dev.txt
+
+COPY src /app/
```

Comment:
Ik zou /app naar /src willen aanpassen, zodat het in lijn is met alle andere apps.

Reply:
Goed punt. Aangepast.